WhateverGreen/kern_rad.hpp

```diff
@@ -269,6 +274,43 @@ class RAD {
 		[RAD::IndexRadeonHardwareX5000] = "__ZN37AMDRadeonX5000_AMDGraphicsAccelerator19populateAccelConfigEP13IOAccelConfig"
 	};
 
+	/**
+	 *  getHWInfo function type
+	 */
+	using t_getHWInfo = int (*)(IOService *accelVideoCtx, void *hwInfo);
+
+	/**
+	 *  getHWInfo wrapping functions used for AppleGVA enable patch
+	 */
+	template <size_t Index>
+	static int populateGetHWInfo(IOService *accelVideoCtx, void *hwInfo) {
```

Comment:
I believe it is not int but rather IOReturn, given the scope.

Reply:
It is IOReturn:

```
__int64 __fastcall AMDRadeonX4000_AMDAccelVideoContext::getHWInfo(__int64 a1, sHardwareInfo *a2)
{
...
if ( !a2 )
    return 0xE00002C2LL;
```